examples/blog-starter/src/app/layout.tsx

```diff
@@ -0,0 +1,66 @@
+import Footer from "@/app/_components/footer";
+import { CMS_NAME, HOME_OG_IMAGE_URL } from "@/lib/constants";
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+
+import "./globals.css";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: `Next.js Blog Example with ${CMS_NAME}`,
+  description: `A statically generated blog example using Next.js and ${CMS_NAME}.`,
+  openGraph: {
+    images: [HOME_OG_IMAGE_URL],
+  },
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  return (
+    <html lang="en">
+      <head>
```

Comment:
I believe this may add be adding a duplicate `<main>` tag. I can fix this later today.

Reply:
@zeevo Thanks for pointing this out!

Reply:
@samcx Fixed it now with latest (squashed) commit 👍